lms/static/sass/views/_shoppingcart.scss

```diff
@@ -1,336 +1,191 @@
 // lms - views - shopping cart
 // ====================
 
+$cart-list-border: 1px solid $border-color-1;
+$hr-border: 1px solid $dark-gray2;
+$input-border: 2px solid $dark-gray2;
+$discount-border: 2px solid $light-gray1;
+$steps-border: 3px solid $light-gray1;
+$light-border: 1px solid $gray-l5;
+$submit-hover-color: #1F8FC2;
+$course-title-color: #4A4A46;
+$shopping-cart-input-color: #666;
+$enrollment-text-color: #9B9B93;
+$contact-support-background-color: #9B9B9B;
+$empty-cart-text-color: #9d9d9d;
+$expiry-border-color: #f0f0f0;
+$card-errors-background: #FFEEF5;
```

Comment:
Nit: could you please order and rename these by purpose (borders, borders colors, background colors, font colors)?